### PR TITLE
Release notes for 1.5 Beta 2

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
Release notes for upcoming release.

### CODE REVIEW

File: changelog.json [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[New] Banner message displayed after successful merge - #5851",
```

File: changelog.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+      "[New] Merge button in branch selector - #5929. Thanks @bruncun!",
+      "[New] Merge conflict resolution flow - #5400",
+      "[Fixed] Single file commit summary now matches behavior of GitHub.com - #6017. Thanks @Daniel-McCarthy!",
+      "[Improved] License and GitIgnore Support - #6024. Thanks @say25!",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Updated license and Git ignore templates for initialising a new repository - #6024. Thanks @say25!",
```

File: changelog.json (hunk 1 above)
Comment (billygriffin) [inline changelog.json]: Isn't this `[Added]` and not new?

File: changelog.json [hunk 3]
```diff
@@ -1,5 +1,15 @@
...
+      "[Improved] Title of merge dialog now includes current branch name - #6027",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Merge dialog displays current branch in header - #6027",
```

File: changelog.json (hunk 1 above)
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Notify users when a merge is successfully completed - #5851",
```

File: changelog.json [hunk 4]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
+      "[New] Merge button in branch selector - #5929. Thanks @bruncun!",
+      "[New] Merge conflict resolution flow - #5400",
+      "[Fixed] Single file commit summary now matches behavior of GitHub.com - #6017. Thanks @Daniel-McCarthy!",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[Improved] Commit summary when committing single file  - #6017. Thanks @Daniel-McCarthy!",
```

File: changelog.json [hunk 5]
```diff
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "1.5.0-beta2": [
+      "[New] Successful merge banner - #5851",
+      "[New] Merge button in branch selector - #5929. Thanks @bruncun!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Allow merging branches directly from branch dropdown - #5929. Thanks @bruncun!",
```

File: changelog.json (hunk 2 above)
Comment (outofambit) [inline changelog.json]:
```suggestion
      "[Improved] Updated license and .gitignore templates for initializing a new repository - #6024. Thanks @say25!",
```

File: changelog.json [hunk 6]
```diff
@@ -1,5 +1,15 @@
...
+      "[New] Merge conflict resolution flow - #5400",
```
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[New] Improved flow for resolving merge conflicts - #5400",
```

Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Added] Resolve merge conflicts through a guided flow - #5400",
```

**Comment by outofambit** [comment]
(note: i'd like to revisit these for the public release)

File: changelog.json (hunk 4 above)
Comment (billygriffin) [inline changelog.json]: I think this should be improved, not fixed, right. It's not a bug I don't think.

```suggestion
      "[Improved] Default commit summary more explanatory and consistent with GitHub.com - #6017. Thanks @Daniel-McCarthy!",
```

File: changelog.json (hunk 5 above)
Comment (shiftkey) [inline changelog.json]:
```suggestion
      "[New] Merge button to perform a merge when switching branches - #5929. Thanks @bruncun!",
```

**Comment by iAmWillShepherd** [comment]
🍎

File: changelog.json [hunk 7]
```diff
@@ -1,5 +1,15 @@
...
+      "[Fixed] Single file commit summary now matches behavior of GitHub.com - #6017. Thanks @Daniel-McCarthy!",
+      "[Improved] License and GitIgnore Support - #6024. Thanks @say25!",
+      "[Improved] Title of merge dialog now includes current branch name - #6027",
+      "[Improved] Merge dialog now includes merge direction - #5930. Thanks @JQuinnie!",
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Clarify the direction of merging branches - #5930. Thanks @JQuinnie!",
```

File: changelog.json [hunk 8]
```diff
@@ -1,5 +1,15 @@
...
+      "[Improved] \"Update From Default Branch\" menu item now shows the branch name - #3018. Thanks @a-golovanov!"
```
Comment (billygriffin) [inline changelog.json]:
```suggestion
      "[Improved] Show the branch name in \"Update From Default Branch\" menu item - #3018. Thanks @a-golovanov!"
```

File: changelog.json (hunk 6 above)
Comment (outofambit) [reply changelog.json]: I think the word `improved` is confusing here. I know we handled conflicts in a manner before so maybe we could say "[New] Merge conflict resolution helper - #5400" instead? or something like that? cc @ampinsk

File: changelog.json (hunk 1 above)
Comment (iAmWillShepherd) [reply changelog.json]: I thought we used `[Added]` things like text editors and code highlighting?

File: changelog.json (hunk 6 above)
Comment (shiftkey) [reply changelog.json]: I like @billygriffin's suggestion here https://github.com/desktop/desktop/pull/6040#discussion_r229402349

Comment (ampinsk) [reply changelog.json]: I like what @billygriffin suggested here!

**Comment by shiftkey** [comment]
This has shipped 🚢

**Pull request merged**